packages/core/src/components/ChoiceList/Choice.jsx

```diff
@@ -19,6 +19,12 @@ export class Choice extends React.PureComponent {
   constructor(props) {
     super(props);
 
+    if (props['inputPlacement'] === 'right') {
+      console.error(
+        `[Deprecated]: 'inputPlacement' is no longer supported and will be removed in a future release, please remove right-to-left <Choice>`
```

Comment:
Tested this and it's working 👍 
I'm wondering if we should add text about it being a React property something like 
```
[Deprecated]: Please remove the React property 'inputPlacement' for the <Choice> component. It is no longer supported and will be removed in a future release.```

Reply:
that sounds good!